src/components/split-button/split-button.scss

```diff
@@ -13,6 +13,10 @@
     }
 }
 
+:host([disabled]) {
+    pointer-events: none;
```

Comment:
I agree. Even though we have done this in many places in Lime Elements (like `limel-button`), this is still a CSS-hack. 

We shouldn't fix functionality bugs with CSS hacks, specially when we are using correct elements in the HTML structure, like links or buttons. We have had similar discussions previously for other components too. 

The problem is that the consumer defines an onClick action on the `host`. So basically, wherever on the host the end-user clicks, the action will be run. But the correct way is that the component should take that action and apply it on the "clickable" element within the component.

In case of `limel-button` this component hosts a `<button` element in itself. But because the `<button` element is physically always as big as the container `host` (`limel-button`), the end user can run the onClick anywhere on the host and the action will run. However, let's think about another component that has a clickable part, and an ornamental or visual part. For instance `limel-shortcut`. The label is located below the "clickable" part (which is an `<a` element in this case). Logically, clicking on the clickable part should run the action not clicking on the label. But when consumers specify an onClick on the host, then the en user can even click on the label to run the action. This is actually a wrong interaction, but since the label and clickable part are so close to each other, it doesn't have a big impact. The impact is only experienced when the component is `disabled` but there is an onClick action on the host.

So, right now `limel-shortcut` also has the exact same problem, that you have solved here. 

Either we should always remember to add 
```scss
:host([disabled]) {
    pointer-events: none;
}
```
to all of such components.

Or find a more sustainable way to handle this in the code.

⚠️ note that: 
1. ` pointer-events: none;` only works for pointers (cursors, finger tips, etc…). It doesn't work for instance on an element that has rabindex and is focusable using keyboard. So pressing <kdb>Enter</kbd> would still run the action! (not here in `limel-split-button` though, since we `disable` both the button and menu trigger. So there is nothing to tab into.
2. some components can have several clickable parts, like this one. We have a menu and a button. Defining onClick on the host is logically weird.
3. the point that Adrian mentioned about the collapsible sections.